tests/integration/routes/profile/patch-profile-test.js

```diff
@@ -15,7 +15,7 @@ var routeOptions = {
   },
   payload: {
     data: {
-      id: 'userid123-profile',
+      id: 'userid123',
```

Comment:
Ah I see. Thanks for the tip. I wasn't aware that account and profile are 2 separate things. Just out of curiosity, could you point me to the code where the profile is handled on account creation?